src/edu/stanford/dlss/was/WasapiFile.java

```diff
@@ -0,0 +1,137 @@
+package edu.stanford.dlss.was;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class corresponding to JSON returned by WASAPI that represents WebdataFile
+ * @see https://github.com/WASAPI-Community/data-transfer-apis/blob/master/ait-implementation/wasapi/implemented-swagger.yaml#L132-L183
+ */
+public class WasapiFile {
+
+  @JsonProperty("account")
+  private int accountId;
+
+  @JsonProperty("checksum")
+  private String checksumsStr;
+
+  @JsonProperty("collection")
+  private int collectionId;
+
+  @JsonProperty("crawl")
+  private int crawlId;
+
+  @JsonProperty("crawl_start")
+  private String crawlStartDateStr;
+
+  private String filename;
```

Comment:
I don't mean to be super pedantic, but could we remove the blank lines between the filename, filetype, locations and size declarations? I like the extra whitespace between the annotated properties, but this is a little too much?